src/windows/functions.cpp

```diff
@@ -393,6 +393,37 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   return MOUNTUTILS_SUCCESS;
 }
 
+BOOL IsDriveLetterEjectable(TCHAR driveLetter) {
```

Comment:
Could be, however we will need to add a lot of v8 specific code to transform values from/to the JS context.

I'm sure there must be an easy Google Test setup using gyp that we can rely on for these kinds of things.